presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,411 @@ Except for the limit on queued queries, when a resource group runs out of a reso
-  new queries will be rejected.
+Sub-groups allow for hierarchical resource allocation, where each sub-group can have its own resource limits and
+queueing policies. A resource group that accepts queries directly is a leaf group, and it executes queries using
+its allocated resources.
 
-* ``hardConcurrencyLimit`` (required): maximum number of running queries.
+The resource groups and associated selection rules are configured by a manager, which is pluggable.
+Presto resource management can be done in two ways:
 
-* ``softMemoryLimit`` (required): maximum amount of distributed memory this
-  group may use before new queries become queued. May be specified as
-  an absolute value (i.e. ``1GB``) or as a percentage (i.e. ``10%``) of the cluster's memory.
+File-Based Resource Management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* ``softCpuLimit`` (optional): maximum amount of CPU time this
-  group may use in a period (see ``cpuQuotaPeriod``) before a penalty will be applied to
-  the maximum number of running queries. ``hardCpuLimit`` must also be specified.
+In a file-based resource manager, configuration information about the resource groups is stored in a JSON file.
+This file contains definitions of all resource groups and the rules to select them.
+The configuration file is loaded and used at the start of the Presto server.
+Any changes to the file require a restart of the Presto server to take effect.
+
+Database-Based Resource Management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In a database-based resource manager, configuration information about the resource groups is stored in a relational
+database. The database contains definitions of all resource groups and the rules to select them.
+Unlike the file-based resource manager, changes to the configuration in the database take effect
+immediately and do not require a restart of the Presto server.
+
+Both methods have their pros and cons. File-based resource management is simpler to set up but less flexible,
+while database-based resource management is more complex to set up but offers more flexibility and dynamic changes.
+
+File Resource Group Manager
+---------------------------
+
+The file resource group manager in PrestoDB is a way to manage resources using a JSON configuration file.
+This file contains definitions of all resource groups and rules for selecting the appropriate resource group
+for a given query.
+
+To set up a file-based resource group manager:
+
+1. Create the file ``etc/resource-groups.properties``.
+
+2. Set the ``resource-groups.configuration-manager`` property to ``file``.
+
+3. Create a JSON file: This file should contain the definitions for the resource groups.
+   Each resource group can specify things like the maximum memory, the maximum queued queries,
+   and the maximum running queries. For information on creating resource group definitions,
+   see ``resources-groups-example.json`` file.
```

Comment:
Suggest you link to the headings in this file where you provide the information the user needs to create the content of the resource-groups.json file.  For example, delete this sentence 
"For information on creating resource group definitions,
   see ``resources-groups-example.json`` file."
   
   and add something like this
   
   ```
   For information on creating resource group definitions,
   see `Resource Group Properties`_. 
   
   For an example of a resource-groups.json file, see `File Resource Group Manager <resource-groups.html#id2>`_. 
```

   
Does this suggestion make sense? I've added a screenshot of how these two suggested sentences look when I build it locally as part of Step 3. 
<img width="765" alt="Screenshot 2023-10-03 at 3 04 37 PM" src="https://github.com/prestodb/presto/assets/7013443/fcaae076-7f93-49ef-8e7c-de88966890c1">

Important note: I know those links aren't formatted the same, but that is how I managed to get them to build successfully as working links. If you can format those links differently that also work, please do so.